main.go

```diff
@@ -24,15 +25,25 @@ func main() {
 		os.Exit(1)
 	}
 
-	// input send text
+	//Form JSON payload to send to Slack
+	var json string
 	args := strings.Join(os.Args[1:], " ")
 	if args == "" {
-		fmt.Fprintln(os.Stdout, "usage: slack-notifier <TEXT>")
-		os.Exit(0)
+		// use stdin as post data
+		if buf, err := ioutil.ReadAll(os.Stdin); err != nil {
+			fmt.Fprintln(os.Stderr, err)
+			os.Exit(1)
+		} else if len(buf) > 0 {
+			json = string(buf)
+		} else {
+			fmt.Fprintln(os.Stdout, "usage: slack-notifier <TEXT>")
+			os.Exit(0)
```

Comment:
But [old source code](https://github.com/wantedly/slack-notifier/pull/7/files#diff-7ddfb3e035b42cd70649cc33393fe32cL31) returns `0`.
Should I change this?